packages/lambdas/scripts/prep-deploy.sh

```
@@ -26,6 +26,10 @@ main() {
    pushd layers/langchain
    npm run prep-deploy
    popd
+
+   pushd layers/analytics-platform
```

Shellcheck (0.11.0)

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)